src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,16 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "countries")
 public class Country implements Cloneable {
```

Comment:
The `Cloneable` interface and its method `clone()` are not necessary for the `Country` class in the context of Hibernate. Hibernate handles entity duplication in its own way. Unless there's a specific requirement for cloning in the project that isn't covered by Hibernate, it's better to remove the `Cloneable` interface and the `clone()` method to avoid confusion.